tidb-configuration-file.md

```diff
@@ -488,6 +488,78 @@ The [`plan cache`](/sql-prepared-plan-cache.md) configuration of the `PREPARE` s
 - Default value: `0.1`
 - The minimum value is `0`; the maximum value is `1`.
 
+## opentracing
+
+Configuration items related to opentracing.
+
+### `enable`
+
++ Enables opentracing to trace the call overhead of some TiDB components. Note that enabling opentracing will cause some performance loss.
++ Default value: `false`
+
+### `rpc-metrics`
+
++ Enables RPC metrics.
++ Default value: `false`
+
+## opentracing.sampler
+
+Configuration items related to opentracing.sampler.
+
+### `type`
+
++ Specifies the type of the opentracing sampler.
++ Default value: `"const"`
++ Value options: `"const"`, `"probabilistic"`, `"rateLimiting"`, `"remote"`
+
+### `param`
+
++ The parameter of the opentracing sampler.
+    - For the `const` type, the value can be `0` or `1`, which indicates whether to enable the `const` sampler.
+    - For the `probabilistic` type, the parameter specifies the sampling probability, which can be a float number between `0` and `1`.
+    - For the `rateLimiting` type, the parameter specifies the number of spans sampled per second.
+    - For the `remote` type, the parameter specifies the sampling probability, which can be a float number between `0` and `1`.
++ Default value: `1.0`
+
+### `sampling-server-url`
+
++ The HTTP URL of the jaeger-agent sampling server.
++ Default value: `""`
+
+### `max-operations`
+
++ The maximum number of operations that the sampler can trace. If an operation is not traced, the default probabilistic sampler is used.
++ Default value: `0`
+
+### `sampling-refresh-interval`
+
++ Controls the frequency of polling the jaeger-agent sampling policy.
++ Default value: `0`
+
+## opentracing.reporter
+
+Configuration items related to opentracing.reporter.
+
+### `queue-size`
+
++ The queue size with which the reporter records spans in memory.
++ Default value: `0`
+
+### `buffer-flush-interval`
+
++ The interval at which the reporter flushes the spans in memory to the storage.
```

Comment:
I think this description provides more details than the Chinese one. If possible, could we update the Chinese description later according to the English one?